hazenlib/tasks/acr_uniformity.py

```diff
@@ -78,73 +78,53 @@ def get_integral_uniformity(self, dcm):
         # Required pixel radius to produce ~1cm2 ROI
         r_small = np.ceil(np.sqrt(100 / np.pi) / self.ACR_obj.dx).astype(int)
         # Offset distance for rectangular void at top of phantom
-        d_void = np.ceil(5 / self.ACR_obj.dx).astype(int)
+        d_void = np.ceil(5 / self.ACR_obj.dy).astype(int)
```

Comment:
please add a description of why it's reasonable to change from x to y coordinate(?) for this calculation

Reply:
I think I changed it to dy because it's describing the vertical void distance from the top of the phantom and it is possible for dx ≠ dy